exchange/exchange_test.go

```diff
@@ -3443,6 +3443,41 @@ func TestUpdateHbPbCatDur(t *testing.T) {
 	}
 }
 
+func TestInsertStoredImpData(t *testing.T) {
+	impsToStoredRequest := make(map[string]ImpExtInfo)
+	impsToStoredRequest["imp_idApn1_1"] = ImpExtInfo{true, []byte(`{"video":{"h":480,"mimes":["video/mp4"]}}`)}
```

Comment:
Please consider using table tests here for ease of readability

Reply:
This is just one scenario here. If there are more scenarios  - I would implement it as a table, agree.

Reply:
This test is gone due to code refactoring